sim/midas/src/main/scala/midas/SynthUnitTests.scala

```diff
@@ -23,7 +23,8 @@ class WithAllUnitTests extends Config((site, here, up) => {
       Module(new ReadyValidChannelUnitTest(timeout = timeout)),
       Module(new CounterTableUnitTest),
       Module(new LatencyHistogramUnitTest),
-      Module(new AddressRangeCounterUnitTest))
```

Comment:
Is there any reason to change this beyond aesthetics?

Reply:
i was commenting lines out to figure out which test was causing FIRRTL to croak, this made that more brainless.